ros_buildfarm/templates/devel/devel_job.xml.em

```diff
@@ -245,6 +256,106 @@ if pull_request:
     'publisher_xunit',
     pattern='catkin_workspace/test_results/**/*.xml',
 ))@
+@[if not pull_request]@
+@(SNIPPET(
+    'publisher_groovy-postbuild',
+    script='\n'.join([
+        '// COLLATE BUILD TEST RESULTS AND EXPORT BUILD HISTORY FOR WIKI',
+        'import jenkins.model.Jenkins',
+        'import hudson.FilePath',
+        '',
+        '@Grab(\'org.yaml:snakeyaml:1.17\')',
+        'import org.yaml.snakeyaml.Yaml',
+        'import org.yaml.snakeyaml.DumperOptions',
+        '',
+        'manager.listener.logger.println("# BEGIN SECTION: Collate test results for wiki.")',
+        '',
+        '// nr of builds to include in history',
+        'final num_build_hist = 5',
+        '',
+        'try {',
+        '  def data = [',
+        '    "history" : []',
+        '  ]',
+        '',
+        '  // gather info on tests of current build',
+        '  def tresult = manager.build.getAction(hudson.tasks.junit.TestResultAction.class)?.result',
+        '  if (tresult)',
+        '  {',
```

Comment:
Nitpick: please use "cuddled" open / close curly braces (as some of the code already does).